src/huggingface_hub/inference/_generated/types/audio_classification.py

```diff
@@ -27,8 +27,10 @@ class AudioClassificationParameters(BaseInferenceType):
 class AudioClassificationInput(BaseInferenceType):
     """Inputs for Audio Classification inference"""
 
-    inputs: Any
-    """The input audio data"""
+    inputs: str
+    """The input audio data as a base64-encoded string. If no `parameters` are provided, you can
```

Comment:
I just realized that `client.audio_classification()` current implementation can take also a local audio file, or a URL pointing to an audio file as input (thanks to the [_open_as_binary()](https://github.com/huggingface/huggingface_hub/blob/c0fd4e0f7519a4e3659c836081cc7e38c0d14b35/src/huggingface_hub/inference/_client.py#L288) context manager). We also handle base64 encoding in the code. This is actually the case for other tasks that take image or audio as input.

While it's quite convenient for users to not worry about file handling or encoding, it makes the documentation of the library not aligned with the current API specs for the `inputs` field.

Reply:
@Wauplin as discussed, this will be addressed in #2561
LGTM :)